### ISSUE TITLE
Map editor menu. Can we use maps list as local maps catalog ?

### ISSUE THREAD

**Issue by reyandme** [Kromster80/kam_remake#329]
Currently we have maps list in MapEd with basic catalog options - different lists for SP/MP maps, sorting by name/number of locs/size and recently added norma/tactic column with small storage/barracks icons respectively.

We have about 300 maps in release version, ~200MP maps, but some players could have up to thousands probably. So better navigating through them could be a nice feature.

Mainly MapEd what designed for mapmakers. But I believe its used by all players too now. Possible purposes of this list:
- view map
- delete map
- move map out of downloads folder
- change/create new map (for mapmakers)
- maybe rename map, same as for replays, in future

There is a conceptional thing in adding more tools to navigate through the list: it can be used to open MP/Campaign map to see, what is covered with FOW. So will help this situations to occur. While if one making his new map he will not need to navigate much - his last opened in maped map will be selected already. 
But I believe MapEd is actually used a lot to see what is covered with FOW already. 

We could add more filtering/sorting options there. 
This could be:
- adding "time last changed" column
- there was an ideas of adding some filtering for coop/special map. Coop maps can be also normal/tactic, while special - does not matter much, because they usually have not much from original KaM gameplay.

Its questionable to add it as separate column with some icon (gold mine: 2 pickaxes = coop of 2 players)/Capital Letter ("C" for coop, "S" for special etc) in it or it could be in the same column as norma/tactic ?
We have to consider, that adding too many of icons or columns can spoil UI though.
- optional: adding some filter checkboxes

@lewinjh , what do you think? We talked on skype about it with @Kromster80 but did find best solution for now.

**Comment by Kromster80** [comment]
TL;DR of my understanding of the issue: MapEd page in main menu (and MapEd itself) are used by players to preview and manage maps. Do we support that there (by adding player-oriented options, alike map types, filters and such) or find a different solution.

I believe that MapEd page was supposed for mapmakers. I guess that mapmakers don't browse for maps that much. So the menu should stay "as is".

I'm not that against adding more columns in to MapEd page in main menu. But then we need to add similar options into Save/Load tab in MapEd. Also we need to be aware of direction we are going with this. We should look into estimating player's needs to manage and filter/sort maps and see where that should fit (perhaps a better pop-up frame in MP for map selection? or better management of DL maps in MP or MapEd menus?)
